BrainPortal/public/javascripts/cbrain.js

```diff
@@ -191,20 +187,19 @@
     //Makes a button set, buttons that are glued together
     loaded_element.find(".button_set").buttonset();
 
-
     loaded_element.find(".button_with_drop_down > div.drop_down_menu").each(function(e) {
       var menu    = $(this);
       var button  = menu.closest(".button_with_drop_down");
       var keep_open = button.attr("data-open");
       if (keep_open !== "true") {
-        menu.hide(); 
+        menu.hide();
       }
     });
 
-    loaded_element.find(".button_with_drop_down > div.drop_down_menu").find(".hijacker_submit_button").click(function(e) { 
-      loaded_element.find(".drop_down_menu:visible").siblings(".button_menu").click();   
-    });
-
+    loaded_element.find(".button_with_drop_down > div.drop_down_menu")
+                  .find(".hijacker_submit_button").click(function(e) {
```

Comment:
Refining the CSS selector here makes the code clearer and voids another `find` call; `.button_with_drop_down > div.drop_down_menu .hijacker_submit_button`